openstack/rhel-8.6-x86_64-large/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we put `devel` in the directory name as well?

Reply:
Done!

Reply:
Oh my, sorry! :( I meant `nightly`. Because now we have half of the images named `devel` and half of them `nightly`. I wanted to use `nightly` everywhere because we use images from nightly composes and not from development composes. Can I ask for one more force-push? I deeply apologize. :(

Reply:
No worries.  Easy fix :)